scripts/pi-hole/js/utils.js

```diff
@@ -104,8 +104,9 @@ function showAlert(type, icon, title, message) {
   }
 }
 
-function datetime(date) {
-  return moment.unix(Math.floor(date)).format("Y-MM-DD [<br class='hidden-lg'>]HH:mm:ss z");
+function datetime(date, html) {
```

Comment:
Is it valid to call this function with one parameter (like `utils.datetime(data.date_added)`) here? I'm sure this is done in some places like `groups.js`.

Reply:
Yeah, it should be fine because `html` won't be false which is the only value I check for.

Reply:
Either way, this in untested like I said in the PR description. I will try to find some time to test it in the next days.